backend/server/auth.go

```diff
@@ -60,7 +60,12 @@ func (s *Server) registerAuthRoutes(g *echo.Group) {
 				}
 
 				var err error
-				user, err = s.store.GetUserByEmail(ctx, login.Email)
+				emptyIdp := ""
+				user, err = s.store.GetUser(ctx, &store.FindUserMessage{
+					Email:                      &login.Email,
+					ShowDeleted:                true,
+					IdentityProviderResourceID: &emptyIdp,
```

Comment:
Instead of passing an empty IdP, can't we make sure if `IdentityProviderResourceID` is not given, we only look for no-IdP users?

Reply:
The `IdentityProviderResourceID` may be these values:
* `nil` for searching all users including bytebase and IdP
* `""` for bytebase users
* `"xxx"` for IdP users

Reply:
> "" for bytebase users

Alright, I suggest we define a constant like `const IdentityProviderResourceIDBytebase = ""`, so it is more clear and no more ad-hoc `emptyIdP` variables.